examples/coap/README.md

```diff
@@ -4,6 +4,21 @@
 
 This application is a work in progress demo of running CoAP with OSCORE/EDHOC security on RIOT-rs.
 
+## Running
+
+* Run on any board with networking, eg. `laze build -b particlex-xenon run`.
```

Comment:
```suggestion
* Run on any board with networking, eg. `laze build -b particle-xenon run`.
```

Reply:
Fixeup'd. (Sorry for the gap in the PR history: I had it as a separate commit, pushed it, squashed, force-pushed, and then found that the push went to the other branch name I use locally since redoing the merge, and is thus not on record.).